src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+
 public class ArrayList<T> implements List<T> {
+
+
+    private static final int def_capacity = 10;
```

Comment:
```suggestion
    private static final int DEFAULT_CAPACITY = 10;
```
Constants should be all uppercase. Also, try to avoid abbreviations, `DEF_CAPACITY` is ambiguos.